app/views/appliances/create.js.erb

```diff
@@ -0,0 +1 @@
+$("#new_appliance_form").replaceWith("<%= j(render @appliance) %>");
```

Comment:
This works, but maybe isn't the best experience since after you create one Appliance, the form disappears from the page and you have to reload to get the form back.

A solution to this is a little tricky since the form is in the same line as the the first two appliances 🤔 

You could try ordering the appliances on the index so the new appliance appears at the end of the list. You couldn't use `replaceWith` but could maybe use [append()](https://api.jquery.com/append/) as long as you identify the last row of Appliances on the page.

This also might not be the user experience you want though, since the new appliance will be at the bottom of the page and then you'd have to scroll to see it 😓 

Perhaps the easiest solution is to create a View partial that is responsible for rendering _all_ appliances as well as the new button/form.

Basically if this was a partial

https://github.com/cbulzoni-appdev/house-manager-v2-rails/blob/68bf8303b48e6c474101d9382cf3f9ac1309624f/app/views/appliances/index.html.erb#L7-L33

then you could put an id on it and re-render this whole partial with the new list of appliances.

This is pretty nitpicky— what do you think?